dash/include/dash/memory/GlobHeapMem.h

```diff
@@ -429,6 +429,23 @@ class GlobHeapMem
     return unit_local_size;
   }
 
```

Comment:
What's wrong with `GlobHeapMem::local_size`? It's fundamentally different from the implementation in any other memory space type, so why should it be centralized?